test/test-style.js

```diff
@@ -128,39 +127,42 @@ function jsonDiff(actual, expected) {
 function testStyle(filename) {
   let hasErrors = false;
   let actual = fs.readFileSync(filename, 'utf-8').trim();
-  let expected = JSON.stringify(JSON.parse(actual), null, 2);
+  /** @type {import('../types').CompatData} */
+  let dataObject = JSON.parse(actual);
+  let expected = JSON.stringify(dataObject, null, 2);
+  let expectedSorting = JSON.stringify(dataObject, orderSupportBlock, 2);
```

Comment:
Just got all the non-chalk changes in here, and then I’d say this PR is ready to go!

Reply:
Moved to #4078